tests/ci/data/profiles/rosa-hcp.yaml

```diff
@@ -4,17 +4,16 @@ profiles:
   channel_group: candidate
   region: "us-west-2"
   cluster:
-    multi_az: true
-    instance_type: "c5.xlarge"
+    multi_az: true # id:75535
```

Comment:
Is it necessary to add the case's number to the yaml file?

Reply:
I can remove them
It was mostly to identify the case based on the cluster config option. But since we have post check, it is not needed anymore

Reply:
updated